src/supporting-data/taxonomy/components/results/TaxonomyCard.tsx

```diff
@@ -0,0 +1,58 @@
+import { Card } from 'franklin-sites';
+import { FC, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { getEntryPath } from '../../../../app/config/urls';
+import TaxonomyView from '../../../../shared/components/entry/TaxonomyView';
+import RenderColumnInCard from '../../../../shared/components/results/RenderColumnInCard';
+import { Namespace } from '../../../../shared/types/namespaces';
+import { TaxonomyAPIModel } from '../../adapters/taxonomyConverter';
+import TaxonomyColumnConfiguration, {
+  TaxonomyColumn,
+} from '../../config/TaxonomyColumnConfiguration';
+
+const mainInfoColumns = [TaxonomyColumn.lineage];
+
+const TaxonomyCards: FC<{
+  data: TaxonomyAPIModel;
+  selected?: boolean;
+  handleEntrySelection: (rowId: string) => void;
+}> = ({ data, selected, handleEntrySelection }) => {
+  const history = useHistory();
+
+  const handleCardClick = useCallback(() => {
+    history.push(getEntryPath(Namespace.proteomes, data.taxonId));
+  }, [history, data.taxonId]);
+
+  return (
+    <Card onClick={handleCardClick}>
+      <section className="result-card">
+        <div className="result-card__left">
+          <input
+            type="checkbox"
+            checked={selected}
+            onClick={(e) => e.stopPropagation()}
+            onChange={() => handleEntrySelection(String(data.taxonId))}
+            data-testid="up-card-checkbox"
+          />
+        </div>
+        <div className="result-card__right">
+          <h5>
+            <TaxonomyView data={data} />
+          </h5>
+          <div className="result-card__info-container">
+            {mainInfoColumns.map((column) => (
+              <RenderColumnInCard
+                type={column}
+                data={data}
+                columnConfig={TaxonomyColumnConfiguration}
```

Comment:
Looking at the API of this now in usage, I'm thinking there's not too much point passing both `type` and `columnConfig` as the full column config where you could just pass, here for example, `TaxonomyColumnConfiguration.get(column)` for the `columnConfig` prop

Reply:
Proposal alternative approach in https://github.com/ebi-uniprot/uniprot-website/pull/319